typeDefs.js

```diff
@@ -393,9 +397,13 @@ const typeDefs = gql`
     uploadFiles(files: [Upload]!): [File]!
     deleteFiles(fileNames: [String]): [String]
     "Pattern Mutations"
-    addPattern(pattern: PatternInput!): PatternResult
+    addPattern(pattern: PatternInput!, upload: Upload): PatternResult
```

Comment:
Please, rename the second argument to the `image`

Reply:
fixed